### ISSUE TITLE
Relative `baseUrl` path in `tsconfig.json` causes `base_dir(...) must be absolute` error

### ISSUE THREAD

**Issue by Phryxia** [SukkaW/rollup-plugin-swc#63]
- Note that in `tsconfig.json`, `baseUrl` can be relative path. [#reference](https://www.typescriptlang.org/tsconfig#baseUrl)
- To reproduce, please checkout [this](https://github.com/Phryxia/rollup-swc-test) repository.
   - It contains minimal reproduction files.
   - If you comment out `baseUrl` in `tsconfig.json`, `npm run build` will emit `bundle.js`
   - If you contain `baseUrl` in `tsconfig.json`, `npm run build` will cause errors like following:

```
> rollupswc-test@1.0.0 build
> rollup --config rollup.config.js


index.ts → bundle.js...
thread '<unnamed>' panicked at /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/swc_ecma_transforms_module-0.179.5/src/path.rs:122:13:
base_dir(`./src`) must be absolute. Please ensure that `jsc.baseUrl` is specified correctly.
```

**Comment by FranciscoKloganB** [comment]
Just need to pass `baseUrl: path.resolve('.')` maybe this could be handled internally by the plugin?

Or import tsconfig from 'tsconfig.json' and pass `baseUrl: path.resolve(tsconfig.baseUrl)`

**Comment by SukkaW** [comment]
That's weird. The issue should have been fixed by #44 and 20de029de538a7ae5e03e86d7a5bf144e25eb487 and has already been released with version `0.10.1`.

**Comment by SukkaW** [comment]
@Phryxia @FranciscoKloganB @admon84

The issue has been fixed and 0.11.1 has been released.
